adding_species_manuscript.tex

```diff
@@ -167,15 +167,11 @@ \section*{Abstract}
 The initial version of \stdpopsim focused on establishing this framework using 
 six well-characterized model species.
 Here, we report on major updates made in the new release of \stdpopsim (version 0.2),
-which includes a significant expansion of the species catalog and substantial additions to simulation capabilities that make it possible to simulate a broader range of species.
+which includes a significant expansion of the species catalog and substantial additions to simulation capabilities.
```

Comment:
it's not yet clear why adding gene conversion makes it possible to simulate more species, so this was confusing, i thought

Reply:
I understood it, and it's a good thought, but I didn't think it'd make sense to the reader at this point.

Reply:
(happy to stick it back in though if someone thinks we should)

Reply:
Since we decided we're keeping the discussion of new features to those that are specific to the species catalog, my thought was to add something to that effect. (I think we say elsewhere that while there are also a bunch of additions for selection, we're explicitly not discussing those here, for instance.) I see your point that it isn't immediately clear with just this added clause. I don't know that it's confusing though, and is clarified somewhat in the next sentence.

Reply:
The section in the paper does a good  in tying the added features to the expanded catalog. We don't necessarily have to explicitly make this point in the abstract. The problem with the statement that @petrelharp removed was that it implied that all (or most) of the features we added serve the purpose of allowing simulation of a wider range of species. In fact, this strictly true only for bacterial recombination. We also have finite site mutation models and gene conversion, which do not strictly allow modeling other species. So, I vote to keep @petrelharp's version here.